### PR TITLE
Port EJBContainer Injection xml tests

### PR DESCRIPTION
**Pull request by olendvcook**
test are currently disabled but in com.ibm.ws.ejbcontainer.injection_fat

### CODE REVIEW

**Comment by olendvcook** [comment]
#build

**Comment by olendvcook** [comment]
#build

File: dev/com.ibm.ws.ejbcontainer.injection_fat/test-applications/EJB3INJSXWeb.war/src/com/ibm/ws/ejbcontainer/injection/xml/web/AdvSLEJBInjectionServlet.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+     * with advanced configurations. <p>
+     *
+     * Also test that the values injected into the fields may be looked
+     * up through both the global namespace and the session context. <p>
+     *
+     * And, all but the first (above) will be repeated, to insure injection
+     * does NOT occur from the pooled state. <p>
+     */
+    @Test
+    public void testSLEJBFldInjection() throws Exception {
```
Comment (tkburroughs) [inline dev/com.ibm.ws.ejbcontainer.injection_fat/test-applications/EJB3INJSXWeb.war/src/com/ibm/ws/ejbcontainer/injection/xml/web/AdvSLEJBInjectionServlet.java]: this test method name is duplicated; need to rename one

File: dev/com.ibm.ws.ejbcontainer.injection_fat/test-applications/EJB3INJSXWeb.war/src/com/ibm/ws/ejbcontainer/injection/xml/web/BasicSLEJBInjectionServlet.java [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+     * with Business Interfaces only. <p>
+     *
+     * Also test that the values injected into the fields may be looked
+     * up through both the global namespace and the session context. <p>
+     *
+     * And, all but the first (above) will be repeated, to insure injection
+     * does NOT occur from the pooled state. <p>
+     */
+    @Test
+    public void testSLEJBFldInjection() throws Exception {
```
Comment (tkburroughs) [inline dev/com.ibm.ws.ejbcontainer.injection_fat/test-applications/EJB3INJSXWeb.war/src/com/ibm/ws/ejbcontainer/injection/xml/web/BasicSLEJBInjectionServlet.java]: this test method name is duplicated; need to rename one

File: dev/com.ibm.ws.ejbcontainer.injection_fat/test-applications/EJB3INJSXWeb.war/src/com/ibm/ws/ejbcontainer/injection/xml/web/BasicSLEJBInjectionServlet.java [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+     * with Business Interfaces only. <p>
+     *
+     * Also test that the values injected into the fields may be looked
+     * up through both the global namespace and the session context. <p>
+     *
+     * And, all but the first (above) will be repeated, to insure injection
+     * does NOT occur from the pooled state. <p>
+     */
+    @Test
+    public void testSLEJBMthdInjection() throws Exception {
```
Comment (tkburroughs) [inline dev/com.ibm.ws.ejbcontainer.injection_fat/test-applications/EJB3INJSXWeb.war/src/com/ibm/ws/ejbcontainer/injection/xml/web/BasicSLEJBInjectionServlet.java]: this test method name is duplicated; need to rename one

File: dev/com.ibm.ws.ejbcontainer.injection_fat/test-applications/EJB3INJSXWeb.war/src/com/ibm/ws/ejbcontainer/injection/xml/web/AdvSLEJBInjectionServlet.java [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+     * with advanced configurations. <p>
+     *
+     * Also test that the values injected into the fields may be looked
+     * up through both the global namespace and the session context. <p>
+     *
+     * And, all but the first (above) will be repeated, to insure injection
+     * does NOT occur from the pooled state. <p>
+     */
+    @Test
+    public void testSLEJBMthdInjection() throws Exception {
```
Comment (tkburroughs) [inline dev/com.ibm.ws.ejbcontainer.injection_fat/test-applications/EJB3INJSXWeb.war/src/com/ibm/ws/ejbcontainer/injection/xml/web/AdvSLEJBInjectionServlet.java]: this test method name is duplicated; need to rename one

**Comment by olendvcook** [comment]
#build

**Comment by olendvcook** [comment]
#build

**Comment by olendvcook** [comment]
#build

**Comment by olendvcook** [comment]
#build

**Comment by olendvcook** [comment]
#build

**Comment by olendvcook** [comment]
#build

**Comment by olendvcook** [comment]
#build

**Comment by olendvcook** [comment]
#build

**Comment by olendvcook** [comment]
#build

**Comment by olendvcook** [comment]
#build

**Comment by olendvcook** [comment]
completely green build, merging

**Pull request merged**